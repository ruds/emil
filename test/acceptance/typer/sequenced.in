(12; "foo");
(12; "foo") : string;
(12; "foo") : bigint;
