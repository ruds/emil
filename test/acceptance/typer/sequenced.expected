@0001:
val it : string
@0002:
val it : string
@0003: ERROR
@0004:
